README.md

```diff
@@ -32,3 +32,44 @@ end
 Documentation can be generated with [ExDoc](https://github.com/elixir-lang/ex_doc)
 and published on [HexDocs](https://hexdocs.pm). Once published, the docs can
 be found at [https://hexdocs.pm/matrix_sdk](https://hexdocs.pm/matrix_sdk).
+
+## Basic Usage
+
+Here is an example to login with username and password.ikyut
```

Comment:
```suggestion
Here is an example to login with username and password:
```

Reply:
That must've been when my cat walked across my keyboard :rofl: